# HG changeset patch
# User Thomas Weber <thomas.weber.mail@gmail.com>
# Date 1211143228 -7200
#      Sun May 18 22:40:28 2008 +0200
# Node ID 08125419efcb261a177f3f34940fd8c914f054db
# Parent  791231dac33344e930465319ec447f3c8ad18ed5
Extend explanation of randn's return value

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2008-05-20  Thomas Weber  <thomas.weber.mail@gmail.com>
+
+	* DLD-FUNCTIONS/rand.cc: Doc fix.
+
 2008-05-20  David Bateman  <dbateman@free.fr>
 
 	* load-save.cc (do_load): Treat non verbose list_only output in
 	the same manner as Fwho.
 	* symtab.h (static void clear_variable_regexp (const
 	std::string&): New method.
 	(static std::list<symbol_record> regexp (const std::string&)): Ditto.
 	(static std::list<symbol_record> regexp_variables (const
diff --git a/src/DLD-FUNCTIONS/rand.cc b/src/DLD-FUNCTIONS/rand.cc
--- a/src/DLD-FUNCTIONS/rand.cc
+++ b/src/DLD-FUNCTIONS/rand.cc
@@ -466,18 +466,19 @@ reset_rand_generator (void *)
 }
 
 DEFUN_DLD (randn, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable Function} {} randn (@var{x})\n\
 @deftypefnx {Loadable Function} {} randn (@var{n}, @var{m})\n\
 @deftypefnx {Loadable Function} {} randn (\"state\", @var{x})\n\
 @deftypefnx {Loadable Function} {} randn (\"seed\", @var{x})\n\
-Return a matrix with normally distributed random elements. The\n\
-arguments are handled the same as the arguments for @code{rand}.\n\
+Return a matrix with normally distributed pseudo-random\n\
+elements having zero mean and variance one. The arguments are\n\
+handled the same as the arguments for @code{rand}.\n\
 \n\
 By default, @code{randn} uses a Marsaglia and Tsang Ziggurat technique to\n\
 transform from a uniform to a normal distribution. (G. Marsaglia and\n\
 W.W. Tsang, 'Ziggurat method for generating random variables',\n\
 J. Statistical Software, vol 5, 2000,\n\
 @url{http://www.jstatsoft.org/v05/i08/})\n\
 \n\
 @seealso{rand, rande, randg, randp}\n\
